src/app/legacy/psammead/psammead-social-embed/src/utilities/index.js

```diff
@@ -9,7 +9,7 @@ export const getProviderName = provider => {
     throw Error("Expected 'provider' to be a string.");
   return {
     instagram: 'Instagram',
-    twitter: 'Twitter',
+    twitter: 'X',
```

Comment:
Should this object key value be X? Or is there some funky legacy stuff happening which requires it to be retainred as `twitter`